orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisQueue.kt

```diff
@@ -229,6 +244,13 @@ class RedisQueue(
       } else {
         try {
           val message = convertToMessage(json, mapper)
+
+          // TODO: AttemptsAttribute could replace `attemptsKey`
+          message.setAttribute(
```

Comment:
Yes, I was about to say I'd like to unify the mechanisms for maxing out retries so we're not doing it two different ways.